nuxeo-tools-hooks/nxtools/hooks/services/github_service.py

```diff
@@ -364,8 +367,13 @@ def activate(self):
         return self.config('active', False)
 
     @property
-    def required_organizations(self):
-        orgas = self.config('required_organizations', [])
-        if orgas:
-            orgas = re.sub(r"\s+", "", orgas, flags=re.UNICODE).split(",")
-        return orgas
+    def whitelisted_private_repositories(self):
+        return self.configlist('whitelisted_private_repositories', [])
+
+    def handle_repository(self, organization, name, private):
+        if not private:
+            return True
+        key = "%s/%s" % (organization, name)
+        if key in self.whitelisted_private_repositories:
+            return True
+        return False
```

Comment:
You could simplifiy:
```suggestion
        return key in self.whitelisted_private_repositories
```